src/VideoUtils.ts

```diff
@@ -68,7 +68,7 @@ export async function getVideoInfo(videoGuids: Array<string>, session: Session,
         let response: AxiosResponse<any> | undefined =
             await apiClient.callApi('videos/' + guid + '?$expand=creator', 'get');
 
-        title = sanitizeWindowsName(response?.data['name']);
+        title = response?.data['name'];
```

Comment:
please restore the way we sanitized filenames as per [this comment](https://github.com/snobu/destreamer/pull/205/files#r470796828)